recipes/tipp/meta.yaml

```diff
@@ -39,16 +41,18 @@ requirements:
     - trf =4.09.1
     - zlib
     - perl
+
 test:
   commands:
     - TIPP_plastid.v2.1.pl -h
     - TIPP_telomere.pl -h
+    - TIPPo.v2.3.pl -h
 
 about:
   home: https://github.com/Wenfei-Xian/TIPP
   license: "GPL-3.0-or-later"
   license_family: GPL3
-  summary: "TIPP: A User-Friendly Tool for De Novo Assembly of Organellar Genomes with HiFi Data"
+  summary: "TIPPo: A User-Friendly Tool for De Novo Assembly of Organellar Genomes with HiFi Data"
```

Comment:
The semantics isn't totally clear to me. Is `TIPPo` a replacement of `TIPP` or should it be a different package? Your current git repo https://github.com/Wenfei-Xian/TIPP is names `TIPP` but the first header in the README.md says `TIPPo`?!

Reply:
Dear @sjanssen2,

Thank you so much for your review. TIPP and TIPPo are essentially the same. When the manuscript was initially submitted, it was named TIPP_Plastid. However, during the review process, the reviewers suggested changes, and the name was updated to TIPPo.

The entire package, including TIPP, TIPPo, and TIPP_plastid, is part of the TIPP system. I specifically included TIPPo to help people easily cite or locate the manuscript.

Thank you again for your valuable feedback, and please let me know if further clarification is needed.

Best regards,
Anupam Gautam